test/run-tests.py

```diff
@@ -533,6 +592,10 @@ def main(args):
                       help='override wasm-interp executable.')
   parser.add_argument('-v', '--verbose', help='print more diagnotic messages.',
                       action='store_true')
+  parser.add_argument('--no-multithreaded', help='Run all tests in series',
+                      action='store_true')
+  parser.add_argument('--stop', help='Stop on first error',
```

Comment:
Mention in the help that on error this enters an interactive mode. I almost want to call it `--stop-interactive`... :)